oak/server/rust/oak_runtime/src/node/mod.rs

```diff
@@ -93,6 +93,7 @@ pub fn create_node(
     application_configuration: &ApplicationConfiguration,
     node_configuration: &NodeConfiguration,
     grpc_configuration: &GrpcConfiguration,
+    signature_table: &SignatureTable,
```

Comment:
FYI a way of making this more flexible (e.g. allowing lookup of signatures from internet) may be to replace this struct with a lookup function (or trait) that given the module hash, returns potential signatures (which the runtime then independently verifies). Then the current implementation may correspond to a lookup function that uses the signature table, but a different one may do HTTP requests or other things to actual return that. I think it would be a nice way of abstracting this behaviour. WDYT? (for another PR of course, but perhaps worth creating an issue to track, if you think it's a good idea? then we can also have a flag to oak_loader to choose which lookup mechanism to use, so we can switch between the two)